batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/StudentDao.java

```diff
@@ -0,0 +1,188 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.utils.DbConnection;
+
+import javax.xml.crypto.Data;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.xml.crypto.Data'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)